CMakeLists.txt

```diff
@@ -20,6 +20,9 @@ cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
 ## building with support for C11 on platforms that default to C99 or C89
 set (CMAKE_C_FLAGS "-std=gnu11 ${CMAKE_C_FLAGS}")
 
+## building with support for C11 on platforms that default to C99 or C89
+set (CMAKE_C_FLAGS "-std=gnu11 ${CMAKE_C_FLAGS}")
+
```

Comment:
I don't think this was supposed to be added a second time?

Reply:
Good catch! I added a commit.